### ISSUE TITLE
IRQ name incomplete in kirunq.txt

### ISSUE THREAD

**Issue by JanClaasSchreiber** [HewlettPackard/LinuxKI#51]
Hello Mark, 

in irqlist I see the below. Please note "60" and "61" for the name...
``` 
   IRQ   node         cpu_list  affinity_mask         name
   231      4      353,355,357  00000000,00000000,0000002a,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000  qla2xxx12_qpair60
   232      4      359,361,363  00000000,00000000,00000a80,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000  qla2xxx12_qpair61
``` 
But in kirunq I see "6" only
``` 
  IRQ Name                                Count      ElpTime    Avg(usec)
  231 qla2xxx12_qpair6                     1331     0.001926     1.446892
  232 qla2xxx12_qpair6                     1476     0.002117     1.434320
``` 
The correct mapping can still be done via the IRQ number.

**Comment by MarkCRay** [comment]
The IRQ name length is hardcoded in the LiKI module.   Changing this can be done, but its a bit complex as it changes the format of the trace data.   There are other ways to fix this up, such as parsing the irq_list file.    Will review and figure out the best way to address this.

**Comment by MarkCRay** [comment]
Fixed in version 7.6 released on 2/24/2023
